tests/integration/test_model.py

```diff
@@ -334,3 +338,73 @@ def test_describe_model(c, training_df):
 
     with pytest.raises(RuntimeError):
         c.sql("DESCRIBE MODEL undefined_model")
+
+
+def test_export_model(c, training_df, tmpdir):
+    with pytest.raises(RuntimeError):
+        c.sql(
+            "EXPORT MODEL not_available_model with "
+            "(model_serde ='pickle',"
+            " location = '/tmp/model.pkl')"
+        )
+
+    c.sql(
+        f"""
+        CREATE MODEL IF NOT EXISTS my_model WITH (
+            model_class = 'sklearn.ensemble.GradientBoostingClassifier',
+            target_column = 'target'
+        ) AS (
+            SELECT x, y, x*y > 0 AS target
+            FROM timeseries
+            LIMIT 100
+        )
+    """
+    )
+    # Happy flow
+    temporary_file = tmpdir + "/pickle_model.pkl"
+    c.sql(
+        "EXPORT MODEL my_model with "
+        "(model_serde ='pickle',"
+        " location = '{}')".format(temporary_file)
+    )
+
+    assert (
+        pickle.load(open(str(temporary_file), "rb")).__class__.__name__
+        == "GradientBoostingClassifier"
+    )
+    temporary_file = tmpdir + "/model.joblib"
+    c.sql(
+        "EXPORT MODEL my_model with "
+        "(model_serde ='joblib',"
+        " location = '{}')".format(temporary_file)
+    )
+
+    assert (
+        joblib.load(str(temporary_file)).__class__.__name__
+        == "GradientBoostingClassifier"
+    )
+    try:
+        # Test only when mlflow was installed
+        import mlflow
```

Comment:
In `pytest`, there is a decorator to make tests skip if a library is not installed. Have a look [here](https://www.scivision.dev/pytest-skip-importerror/).
I think that would be the cleanest solution. In this case, you would need to create a single function for only the mlflow test.

Reply:
Amazing, this is new to me, thanks